test/e2e/tests/transaction/send-eth.spec.js

```diff
@@ -154,18 +150,17 @@ describe('Send ETH', function () {
       await withFixtures(
         {
           fixtures: new FixtureBuilder().build(),
-          ganacheOptions: {
-            ...defaultGanacheOptions,
+          networkOptions: {
```

Comment:
this spec is using anvil! When we need to pass custom network options, we can pass them and we can omit the default. 
I've name the options with a generic name `networkOptions`